crates/solvers/src/tests/paraswap/market_order.rs

```diff
@@ -148,7 +148,7 @@ async fn sell() {
     ])
     .await;
 
-    let engine = tests::SolverEngine::new("paraswap", paraswap::config(&api.address)).await;
+    let engine = tests::SolverEngine::new("paraSwap", paraswap::config(&api.address)).await;
```

Comment:
also no need to change this (I think we at some point decided to have all solver names lower case).

Reply:
Reverted